chore: update readme and variables naming

cjoecker opened the pull request:
I used the package and works very smoothly!

I updated the Readme and variables names from my experience installing it. I hope it is now more clear for people installing it.

juhanakristian commented on README.md:
```diff
@@ -49,60 +62,69 @@ authenticator.use(microsoftStrategy);
 
 See [Microsoft docs](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-auth-code-flow) for more information on `scope` and `prompt` parameters.
 
+### Organisations without multitenant
```
I this could be more clear. Maybe something like "Applications with single-tenant authentication" but it's up to you 🙂

cjoecker replied:
You are right, change it ;)

juhanakristian commented on src/index.ts:
```diff
@@ -52,9 +52,9 @@ export class MicrosoftStrategy<User> extends OAuth2Strategy<
   constructor(
     {
-      clientID,
+      clientId,
```
I initially chose to follow the remix-auth conventions with the variable names, but in hindsight, it's probably better to use the Microsoft conventions. 👍

juhanakristian reviewed (commented):
Thanks for the PR 👍 Overall, very good changes and I agree with changing the variable names to match the Microsoft conventions. I left one comment on the header of the multitenant section of the README. 

As these are breaking changes, I'll need to bump the major version of the package.

Thanks again 🙏

cjoecker commented:
I'm the one who needs to thank you for creating the library! I added some small changes to the PR and implemented your suggestion. You can give it a check.
